### PR TITLE
docs: migrate the pages markdown files into pages folder

### PR DESCRIPTION
**Pull request by afzal442**
- Related to #483 
- Part of #480

### CODE REVIEW

**Comment by afzal442** [comment]
Hi, I think it is ready to be reviewed.

cc @Souvikns @KhudaDad414

File: docs/pages/glee-template.md
Comment (KhudaDad414) [inline docs/pages/glee-template.md:1]: I am not sure where this page fits in the documentation.
before making changes to the structure. I suggest you open a discussion and write down the things that we discussed. plus a blue-print of the final documentation. and how the documentation should be navigated.

**Review by KhudaDad414 — CHANGES_REQUESTED** [review]

File: docs/pages/glee-template.md
Comment (afzal442) [reply docs/pages/glee-template.md:1]: Thanks for your suggestion. I have raised an [issue](https://github.com/asyncapi/glee/issues/481) on auto-deployment using gh action. If Animesh is not able to proceed with it, I will create a PR to resolve that next week. 
once it's get merged, we'll be able to view pages rendered on asyncapi website like below and we can customise those if required. 
image

Also I will be opening a discussion on that taking into consideration of glee v3.
WDYT? cc @Souvikns let's merge this for now.

Comment (afzal442) [reply docs/pages/glee-template.md:1]: @KhudaDad414 we are considering `/docs/pages/*.md` files to be displayed for the website. What do you think about it now?

Comment (KhudaDad414) [reply docs/pages/glee-template.md:1]: left some comment here: https://github.com/asyncapi/glee/pull/564
lets do this 💪

**Review by KhudaDad414 — APPROVED** [review]
LGTM

**Pull request merged**